index.html

```diff
@@ -7,6 +7,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 <body>
+<div id="title" style="text-align: center">
+    <h2>DEAD AND WOUNDED</h2>
```

Comment:
Use text-transform instead of typing in all caps
```suggestion
    <h2 style="text-transform: uppercase">Dead and Wounded</h2>
```